Assignment one

connor1484 opened the pull request:
Assignment one pull request

connor1484 commented:
Assignment one done

smarks commented:
in your root directory put a file called .gitignore

files containing the patterns below are excluded from Git

.class
target/

!.mvn/wrapper/maven-wrapper.jar

Mac OS

.DS_Store

STS

.apt_generated
.classpath
.factorypath
.project
.settings
.springBeans

IntelliJ IDEA

.idea
*.iws
*.iml
*.ipr

NetBeans

nbproject/private/
build/
nbbuild/
dist/
nbdist/
.nb-gradle/

smarks commented on src/main/java/edu/connor/advancedjava/Person.java:
```diff
@@ -0,0 +1,52 @@
+import java.time.LocalDate;
+import java.time.Year;
+import java.time.Period;
+
```
javadoc

smarks commented on src/main/java/edu/connor/advancedjava/Student.java:
```diff
@@ -0,0 +1,26 @@
+
+    public static void main(String[] args) {
+
+        String name = "Connor";                             //name of student
```
when I comment code here are the rules I have for myself
only comment lines that are hard to understand / long / confusing or might do something unexpected. 
if I end up writing lots of comments I fix my code.

smarks commented on src/test/java/edu/connor/advancedjava/PersonTest.java:
```diff
@@ -0,0 +1,61 @@
+     *
+     * @param testName name of the test case
+     */
+    @Before
```
// double slash for single line comments 
/* 
* multiple comments 
*/

smarks commented on src/test/java/edu/connor/advancedjava/PersonTest.java:
```diff
@@ -0,0 +1,61 @@
+
+        //testing result of expected age
+        age = student.getAge();
+        assertEquals(38, age);
```
always use the form of assert that takes a string argument first and use that string to describe the test

smarks commented:
have you completed all the changes I requested ?

connor1484 commented:
oh sorry @smarks, I did not. I thought you said not to worry about them for the first assignment.  I made sure to do them for assignment 2.  I can update them after work today and push the changes up though if you want.

connor1484 commented:
Hi @smarks 
Sorry about the confusion. I've updated my code and deleted the extra files.  Let me know if there are any other issues.

Thanks,
Connor
